missionbit/settings.py

```diff
@@ -252,5 +252,12 @@
 
 
 CELERY_BEAT_SCHEDULE = {
-
+    'users_contacts_sync': {
+        'task': 'staff.tasks.add_salesforce_contacts_to_postgres',
+        'schedule': crontab()
+    },
+    'userprofile_sync': {
+        'task': 'staff.tasks.sync_userprofile_data_with_salesforce_data',
+        'schedule': crontab()
+    }
```

Comment:
Once a minute is a bit much as we're fetching every single contact in Salesforce and doing at least as many queries in Postgres, I wouldn't go any more frequent than hourly

Reply:
Yes I'll put a comment that it needs to be changed, I was thinking that I'd do these two once every 24 hours... it's only at 1 minute for testing purposes right now.

Reply:
For testing adding a button to a staff page to trigger a sync might make sense